sw/host/opentitantool/src/command/bootstrap.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use erased_serde::Serialize;
+use humantime::parse_duration;
+use std::any::Any;
+use std::path::PathBuf;
+use std::time::Duration;
+use structopt::StructOpt;
+
+use opentitanlib::app::command::CommandDispatch;
+use opentitanlib::bootstrap::{Bootstrap, BootstrapOptions, BootstrapProtocol};
+use opentitanlib::io::spi::SpiParams;
+use opentitanlib::transport::{Capability, Transport};
+
+/// Bootstrap the target device.
+#[derive(Debug, StructOpt)]
+pub struct BootstrapCommand {
+    #[structopt(flatten)]
+    params: SpiParams,
+    #[structopt(
+        short,
+        long,
+        possible_values = &BootstrapProtocol::variants(),
+        case_insensitive = true,
+        default_value = "primitive",
+        help = "Bootstrap protocol to use"
+    )]
+    protocol: BootstrapProtocol,
+    #[structopt(long, parse(try_from_str=parse_duration), help = "Duration of the reset delay")]
```

Comment:
Doesn't structopt pick up help strings from /// comments?

Reply:
It does, although, I've been pretty consistent in using the `help` argument to supply the help text.  I don't have a particular preference except that using `help` feels more explicit to me.